utils.go

```diff
@@ -102,3 +107,815 @@ func matchRegexSlice(exprs []regex.EnryRegexp, str string) bool {
 
 	return false
 }
+
+// IsGenerated returns whether the file with the given path and content is a
```

Comment:
Maybe it would be better to move other parts of added code to other file ex. `generated.go`? Leaving just this function here otherwise this file becomes too large imho

Reply:
Yes all the regexp and other logic to do the matching should go in `data` even if is not generated we already have the [`isTest`](https://github.com/go-enry/go-enry/blob/master/data/test.go) regexps.